cmd/project.go

```
@@ -821,6 +821,39 @@ func getConfigTemplateDir(cmd *cobra.Command) (string, bool, error) {
 	return dir, false, nil
 }
 
+// Helper to fetch projects and prompt user to select one. Returns selected project ID or empty string.
+func selectProject(ctx context.Context, logger logger.Logger, apiUrl, apikey string, prompt string) string {
```

golangci-lint (1.64.8)

func `selectProject` is unused

(unused)